### PR TITLE
Provide info that you do not need to manually register an action before using it

### PR DESCRIPTION
**Pull request by binarygit**

### CODE REVIEW

**Comment by adrianthedev** [comment]
My inclination is to just remove this section. If a new person comes on, they would wonder why do we need to specify that you "don't need to add it in a second place"...
Does that make sense? WDYT?

**Comment by binarygit** [comment]
True. I was thinking about all the people who use it currently. How do we tell them that now they can remove it from the actions method?

**Comment by adrianthedev** [comment]
The way we do it now is throught the release notes, release video, and upgrade guide. 
Let's add it to `upgrade.md`.

**Comment by binarygit** [comment]
Ahhh. Got it

**Comment by binarygit** [comment]
Could you take a look and tell me if I did it correctly?
I don't know which beta version avo is in, and it looks like every entry is preceded by `::option` so I also used that there.

**Comment by adrianthedev** [comment]
It should go up to the top saying something like "Upgrade from 3.10 to 3.11"
And then a smaller `h` like `### You can remove the ...`" and some body of text below to explain what and why.

**Comment by Paul-Bob** [comment]
Let's also add a message that this warning is not applicable for versions bigger than the next version (3.10.7)
https://docs.avohq.io/3.0/customizable-controls.html#action

**Review by adrianthedev — APPROVED** [review]

**Comment by Paul-Bob** [comment]
> Let's also add a message that this warning is not applicable for versions bigger than the next version (3.10.7) https://docs.avohq.io/3.0/customizable-controls.html#action

@binarygit can you please address [this comment](https://github.com/avo-hq/docs.avohq.io/pull/242#issuecomment-2232991099) so we can merge?

**Comment by binarygit** [comment]
Oops. I just pushed a commit @Paul-Bob adding the note

File: docs/3.0/customizable-controls.md [hunk 1]
```diff
@@ -202,6 +202,8 @@ action Avo::Actions::PublishPost, color: :fuchsia, icon: "heroicons/outline/eye"
 :::
 
 :::warning
+Note: This warning is not applicable for versions greater than 3.10.7
```
Comment (Paul-Bob) [inline docs/3.0/customizable-controls.md:205]:
```suggestion
```

File: docs/3.0/customizable-controls.md [hunk 2]
```diff
@@ -202,6 +202,8 @@ action Avo::Actions::PublishPost, color: :fuchsia, icon: "heroicons/outline/eye"
 :::
 
 :::warning
```
Comment (Paul-Bob) [inline docs/3.0/customizable-controls.md:204]:
```suggestion
:::warning WARNING (**NOT** applicable for versions greater than <Version version="3.10.7" />)
```

**Review by Paul-Bob — APPROVED** [review]
Tweaks

**Pull request merged**